mobile-app/app/screens/AppNavigator/screens/Loans/LoansNavigator.tsx

```diff
@@ -63,6 +71,21 @@ export function LoansNavigator (): JSX.Element {
               text={translate('screens/LoansScreen', 'Loans') + ' (Beta)'} // TODO: remove beta from title
               containerTestID={headerContainerTestId}
             />
+          ),
+          headerRight: () => (
```

Comment:
currently it is also displaying when we are in the `empty vault screen`, maybe can set header in loan screen?

Reply:
It's ok, extra entrypoint is not an issue